source/enetstuff.lua

```diff
@@ -0,0 +1,117 @@
+
+
+local EnetHandler = {}
+
+local server
+local client
+
+local timerHostSendInterval = 0.05
+local timerHostSendTimer = timerHostSendInterval
+
+local timerClientSendInterval = 0.05
+local timerClientSendTimer = timerClientSendInterval
+
+function EnetHandler.createHost()
+-- called by menu
+
+	server = sock.newServer(hostIPAddress, 22122)
+	
+    -- Called when receiving a message of type "connect"
+    server:on("connect", function(data, client)
+        -- Send a message of type "welcome" back to the connected client
+		client:send("welcome", client:getConnectId())
+		
+		newLander = Lander.create()
+		newLander.connectionID = client:getConnectId()
+		table.insert(garrLanders, newLander)
+	end)
+	
+	server:on("clientdata", function(lander, clientInfo)
+		-- match the incoming lander object
+		for k,v in pairs(garrLanders) do
+			if v.connectionID == landerObject.connectionID then

```

Comment:
lander insteadof landerObject

Reply:
> should this be lander instead of landerObject?

I believe so yes. Also couldn't the connectionID be used as index for the garrLanders table? that would get rid of the extra pairs loop.